styles/globals.css

```diff
@@ -1152,7 +1145,80 @@ select {
   border-radius: 0px 0px 7px 7px;
 }
 
-/*Mentor CSS*/
+/*MentorCard CSS starts*/
+.mentorLists {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 30px;
+  gap: 20px;
+  justify-content: center;
+}
+.card-container {
+  margin-bottom: 2rem;
+  width: 18.6vw;
+  height: 100%;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.12);
+  transition: all 0.3s ease;
+}
+
```

Comment:
move this css , All css in mentor card file... you can create mentorCard.css in styles folder